components/apps_form/apps_form_component.test.tsx

```diff
@@ -219,7 +219,7 @@ describe('AppsFormComponent', () => {
                     <AppsForm {...props}/>
                 </Provider>,
             );
-            expect(wrapper.find(Modal.Body).find('.react-select__single-value').text()).toEqual('Option3');
+            expect(wrapper.find(Modal.Body).find('div.react-select__single-value').text()).toEqual('Option3');
```

Comment:
Emotion adds a wrapper around the actual div that has the same classNames as the actual div containing the value, so the test had to become more specific.